src/main/java/kekztech/KekzCore.java

```diff
@@ -65,19 +65,18 @@ public void onMissingMapping(FMLMissingMappingsEvent event) {
         List<FMLMissingMappingsEvent.MissingMapping> missingMappings = event.get();
 
         // intentionally not a static final field to save a bit of ram.
-        Set<String> removedBlocks = ImmutableSet.of(
-                "kekztech_tfftcasingblock_block",
-                "kekztech_tfftmultihatch_block",
-                "kekztech_tfftstoragefieldblock1_block",
-                "kekztech_tfftstoragefieldblock2_block",
-                "kekztech_tfftstoragefieldblock3_block",
-                "kekztech_tfftstoragefieldblock4_block",
-                "kekztech_tfftstoragefieldblock5_block");
+        Set<String> removedStuff = ImmutableSet.of(
+                MODID + ":kekztech_tfftcasingblock_block",
+                MODID + ":kekztech_tfftmultihatch_block",
+                MODID + ":kekztech_tfftstoragefieldblock1_block",
+                MODID + ":kekztech_tfftstoragefieldblock2_block",
+                MODID + ":kekztech_tfftstoragefieldblock3_block",
+                MODID + ":kekztech_tfftstoragefieldblock4_block",
+                MODID + ":kekztech_tfftstoragefieldblock5_block");
 
         for (FMLMissingMappingsEvent.MissingMapping mapping : missingMappings) {
-            if (mapping.type == GameRegistry.Type.BLOCK) {
-                if (removedBlocks.contains(mapping.name)) mapping.ignore();
-            }
+            if (removedStuff.contains(mapping.name)) mapping.ignore();
+            else mapping.warn(); // we don't know what happened. probably warn the user.
```

Comment:
This would probably override what other mods did in their event handlers

Reply:
.get() *theoretically* should only return mappings related to current mod only, but ok.

Reply:
Hmm yeah, the only situation this would be a problem would be if some other mod used getAll and then modified the kekztech entries while it is loaded, it's probably fine then